content/docs/developer/graphql_api.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "GraphQL API"
+description = "Integrating with Open mSupply using the GraphQL API"
+date = 2024-06-27T18:20:00+00:00
+updated = 2024-06-27T18:20:00+00:00
+draft = false
+weight = 7
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
++++
+
+<style>
```

Comment:
not strictly necessary.. but for the developer page I thought it neat to have dark mode imagery.